src/app/shell/details/details-tabs/achievements/achievements.component.ts

```diff
@@ -52,6 +52,7 @@ export class AchievementsComponent implements OnInit {
   }  
 
   ngOnDestroy(): void {
+    this.destroy$.next(true);
```

Comment:
it's of course up to you, but it would be also a good idea to check such places in the code and add this row.

Reply:
I have checked, it's the only place

Reply:
oh ok, thank you for the update